djangocms_moderation/admin_actions.py

```diff
@@ -77,6 +80,7 @@ def delete_selected(modeladmin, request, queryset):
 
 
 delete_selected.short_description = _("Remove selected")
+delete_selected.__name__ = 'remove_selected'
```

Comment:
Why has the action name been changed, Django calls this delete_selected? Is this Dj2.2 related?

Reply:
Yes this django 2.2 related. There is already one action named `delete_selected` and it complains with an error if you try to have the same name twice.